processor/k8sattributesprocessor/metadata.yaml

```diff
@@ -94,6 +94,10 @@ resource_attributes:
     description: Container ID. Usually a UUID, as for example used to identify Docker containers. The UUID might be abbreviated. Requires k8s.container.restart_count.
     type: string
     enabled: false
+  container.image.id:
+    description: Container image digest. Requires container.id (??or k8s.container.name??).
```

Comment:
I believe either would work, are you seeing otherwise?

Reply:
I think you're right (either will work).  I forgot to edit this before submitting the PR.  Apologies for that.